src/Arcus.BackgroundJobs.DataBricks/DatabricksJobMetrics.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Arcus.Security.Core;
+using CronScheduler.Extensions.Scheduler;
+using GuardNet;
+using Microsoft.Azure.Databricks.Client;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Arcus.BackgroundJobs.Databricks
+{
+    /// <summary>
+    /// Representing a background job that repeatedly queries the Databricks instance for finished job runs and writes the report as a metric.
+    /// </summary>
+    public class DatabricksJobMetrics : IScheduledJob
```

Comment:
Let's be a bit more explicit here maybe?
```suggestion
    public class DatabricksJobMetricsJob : IScheduledJob
```